mne/epochs.py

```diff
@@ -1556,13 +1556,10 @@ def equalize_event_counts(self, event_ids, method='mintime', copy=True):
             If 'truncate', events will be truncated from the end of each event
             list. If 'mintime', timing differences between each event list
             will be minimized.
-        copy : bool
-            If True, a copy of epochs will be returned. Otherwise, the
-            function will operate in-place.
```

Comment:
That depends - I don't understand what about the current behavior is insufficient. It seems to do right what we want it to do with the method.

Another issue is what to do with the function. I think it might be worthwhile to give the function a copy parameter (defaulting to the current behavior, but allowing to pick in-placed or copying).